quartz/util/path.ts

```diff
@@ -161,8 +161,15 @@ export function pathToRoot(slug: FullSlug): RelativeURL {
   return rootPath as RelativeURL
 }
 
-export function resolveRelative(current: FullSlug, target: FullSlug | SimpleSlug): RelativeURL {
-  const res = joinSegments(pathToRoot(current), simplifySlug(target as FullSlug)) as RelativeURL
+export function resolveRelative(
```

Comment:
I think we do resolve folder elsewhere?

iirc it might be in ofm.ts instead of here. Util should produce as little change as possible.

Reply:
I couldn't find any other place where this gets done. And this function gets called directly from a component, so there aren't any other layers in between where this could be done?
I can move the logic to the component.. but I have a feeling this will have to be used also in other places?